buidl/hd.py

```diff
@@ -377,7 +377,7 @@ def from_mnemonic(cls, mnemonic, password=b"", path="m", testnet=False):
         # normalize in case we got a mnemonic that's just the first 4 letters
         normalized_words = []
         for word in words:
-            normalized_words.append(WORD_LIST[WORD_LOOKUP[word]])
+            normalized_words.append(BIP39.normalize(word))
```

Comment:
This comment predates this PR, but I noticed we are doing `for word in words:` 2x here, once above for the index and once down here to normalize the words. I think this loop should be done 1x in the same place.